src/components/offer-card/OfferCard.jsx

```diff
@@ -0,0 +1,39 @@
+import AppCard from '~/components/app-card/AppCard'
+import OfferAvatarAndRating from '~/components/offer-card/offer-avatar-and-rating/OfferAvatarAndRating'
+import OfferDetails from '~/components/offer-card/offer-details/OfferDetails'
+import OfferActions from '~/components/offer-card/offer-actions/OfferActions'
+
+const OfferCard = ({
+  imgSrc,
```

Comment:
Too many props. Can you just pass the item?

Reply:
Done.